cpp/include/cudf/scalar/scalar.hpp

```diff
@@ -330,8 +330,7 @@ class fixed_point_scalar : public scalar {
                      bool is_valid                       = true,
                      rmm::cuda_stream_view stream        = rmm::cuda_stream_default,
                      rmm::mr::device_memory_resource* mr = rmm::mr::get_current_device_resource())
-    : scalar{data_type{type_to_id<T>(), _type.scale()}, is_valid, stream, mr},
-      _data{numeric::scaled_integer<rep_type>{value}.value}
+    : scalar{data_type{type_to_id<T>(), value.scale()}, is_valid, stream, mr}, _data{value.value()}
```

Comment:
I'm curious: Do we have a naming convention that describes when leading underscores are to be used in member names like `_data`?

Reply:
No mentioned of it in the dev guide (https://github.com/rapidsai/cudf/pull/6977) but I have noticed in many classes it is the pattern for private members.